battles/forms.py

```diff
@@ -34,6 +36,12 @@ def clean(self):
             self.cleaned_data['pokemon_3'].lower().strip()
         ]
 
+        for pokemon in pokemons:
+            if not exists(pokemon):
+                raise forms.ValidationError(
+                    'We couldnt find "{}". Please, check if you wrote it correctly.'.format(pokemon)
```

Comment:
@pamella, you could use [f-strings](https://realpython.com/python-f-strings/) in here.
It's not something you need to change, but just a tip for future references :)

Reply:
Done!

Reply:
Awesome!